.gitignore

```diff
@@ -2,4 +2,4 @@
 dist
 node_modules
 examples/example-list.js
-_site
```

Comment:
This line is necessary if you build the website locally, it prevents git from syncing the website build folder.